test/precompile.jl

```diff
@@ -956,6 +956,17 @@ precompile_test_harness("code caching") do dir
         use_stale(c) = stale(c[1]) + not_stale("hello")
         build_stale(x) = use_stale(Any[x])
 
+        # bindings
```

Comment:
tests for this code path are mostly in rebinding.jl. I don't really mind it being here, but I don't fully know yet what distinguishes this case from the ones already tested

Reply:
Thanks, I hadn't found that yet. Likewise I'd have no objection to moving these tests to rebinding.jl.

Reply:
One potential difference: from a quick scan, the tests in rebinding.jl seem to define the function in the same module in which it's precompiled. This test defines `StaleA.fib()` but defers its precompilation until `StaleB`.